cmd/oras/pull.go

```diff
@@ -84,6 +85,7 @@ Example - Pull all files with concurrency level tuned:
 
 	cmd.Flags().BoolVarP(&opts.KeepOldFiles, "keep-old-files", "k", false, "do not replace existing files when pulling, treat them as errors")
 	cmd.Flags().BoolVarP(&opts.PathTraversal, "allow-path-traversal", "T", false, "allow storing files out of the output directory")
+	cmd.Flags().BoolVarP(&opts.skipSubject, "skip-subject", "", false, "don't recursively pull subject manifest")
```

Comment:
Can you follow #681 and use `--include-subject`?

Consider this scenario:
1. A user runs `oras discover` to find artifacts.
2. That user would run `oras pull` to pull the attached artifacts multiple times.

In this scenario, the user would always skip the subject.